pyproject.toml

```diff
@@ -1,3 +1,75 @@
+[build-system]
+requires = ["setuptools>=61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "deepsensor"
+version = "0.4.2"
+authors = [
+    {name = "Tom R. Andersson", email="tomandersson3@gmail.com"},
+]
+description = "A Python package for modelling xarray and pandas data with neural processes."
+readme = "README.md"
+license = {text="MIT"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent"
+]
+requires-python = ">=3.8"
+dependencies = [
+    "backends",
+    "backends-matrix",
+    "dask",
+    "distributed",
+    "gcsfs",
+    "matplotlib",
+    "neuralprocesses>=0.2.6",
```

Comment:
Regarding https://github.com/alan-turing-institute/deepsensor/commit/c5b45d91cf6faee975f9bd9f19a5331655030059, I don't think git's merge will pull that change through, since this isn't a fork of the requrements.txt file. I think you'll have to sync on top of https://github.com/alan-turing-institute/deepsensor/commit/c5b45d91cf6faee975f9bd9f19a5331655030059 to avoid undoing it.